### DISCUSSION TITLE
Connecting CS50 Python to Github for problemsets

### DISCUSSION THREAD

**Discussion by AIbagger** [community/community#66989] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

dear fellow students,

I am trying to connect my course account (CS50 Python from harvard) with the github software for submitting assigments and the pairing process does not let me connect. I get this error:

failed to give pull permissions to repo [403] b'{"message":"API rate limit exceeded for user ID 20504399. If you reach out to GitHub Support for help, please include the request ID E45A:2701:1472FAA:29C967E:65019CEB.","documentation_url":"[https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting"}](https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting%22%7D)'

**Comment by seyLu** [comment]
Hi @AIbagger,

[The rate limit for GitHub API requests resets every hour](https://docs.github.com/en/rest/overview/resources-in-the-rest-api?apiVersion=2022-11-28#rate-limiting). If you encounter this error, please try again in an hour or so.
